### PR TITLE
upgrade embeddings api to version 2 for  volcengine

### PR DESCRIPTION
**Pull request by mrichie**
https://www.volcengine.com/docs/82379/1263482

用户前往火山方舟的模型推理页面建立Endpoint，Endpoint为请求接入的入口，绑定接入方身份，限流，计费以及模型编排等信息，不再支持通过模型名+版本访问。

Thank you for contributing to LangChain!

- [ ] **PR title**: "package: description"
  - Where "package" is whichever of langchain, community, core, experimental, etc. is being modified. Use "docs: ..." for purely docs changes, "templates: ..." for template changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [ ] **PR message**: ***Delete this entire checklist*** and replace with
    - **Description:** a description of the change
    - **Issue:** the issue # it fixes, if applicable
    - **Dependencies:** any dependencies required for this change
    - **Twitter handle:** if your PR gets announced, and you'd like a mention, we'll gladly shout you out!

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

### CODE REVIEW

File: libs/community/langchain_community/embeddings/volcengine.py [hunk 1]
```diff
@@ -24,19 +24,16 @@ class VolcanoEmbeddings(BaseModel, Embeddings):
     host: str = "maas-api.ml-platform-cn-beijing.volces.com"
     """host
     learn more from https://www.volcengine.com/docs/82379/1174746"""
+    
     region: str = "cn-beijing"
     """region
     learn more from https://www.volcengine.com/docs/82379/1174746"""
 
-    model: str = "bge-large-zh"
```
Comment (isahers1) [inline libs/community/langchain_community/embeddings/volcengine.py:31]: self.model is referenced on line 116 I believe, I am not sure we can delete it as a param.

Comment (isahers1) [reply libs/community/langchain_community/embeddings/volcengine.py:31]: lgtm other than this, @baskaryan

**Comment by ccurme** [comment]
Hi @mrichie, can you comment on the breaking change here and whether support for users on v1 is a concern?

**Comment by efriis** [comment]
closing for no response - let us know on the above and we can reopen!

**Pull request closed without merging**